### ISSUE TITLE
Changing the namespace in LPython and LFortran

### ISSUE THREAD

**Issue by namannimmo10** [lcompilers/lpython#403]
Right now as you can see,

https://github.com/lcompilers/lpython/blob/1d85426255559f372822da54470cce669c7090bb/src/lpython/semantics/python_ast_to_asr.cpp#L29

For the Python frontend, the current namespace is not suitable.  We should make it `LCompilers::LPython`.

**Comment by Thirumalai-Shaktivel** [comment]
Exactly, We were thinking the same. I will do it, once I finish the work on `new-parser`.
Thanks for opening this...

**Comment by certik** [comment]
Yes, LPython should eventually have no mention of Fortran at all.
